programs/dibio.c

```diff
@@ -12,7 +12,8 @@
 /*-*************************************
 *  Includes
 ***************************************/
-#include "util.h"           /* Compiler options, UTIL_GetFileSize, UTIL_getTotalFileSize */
+#include "platform.h"       /* Compiler options */
```

Comment:
Since `platform.h` is already included at top of `util.h`, is there any benefit at including it again in each user source file of `util.h` ?